rllib/agents/ppo/ddppo.py

```diff
@@ -158,12 +157,6 @@ def validate_config(self, config):
         # setting.
         super().validate_config(config)
 
-        # Error if run on Win.
-        if sys.platform in ["win32", "cygwin"]:
```

Comment:
is there a reason why you removed this?

Reply:
Yeah, it's actually running on Win now, at least with a newer version of torch. I tested this directly on a Win machine.

Reply:
cool cool